services/datamanager/builtin/builtin.go

```diff
@@ -77,6 +77,22 @@ type dataCaptureConfig struct {
 	Disabled           bool              `json:"disabled"`
 	RemoteRobotName    string            // Empty if this component is locally accessed
 	Tags               []string          `json:"tags"`
+	CaptureDirectory   string            `json:"capture_directory"`
+}
+
+func (c *dataCaptureConfig) Equals(other *dataCaptureConfig) bool {
+	return c.Name == other.Name &&
+		c.Model == other.Model &&
+		c.Type == other.Type &&
+		c.Method == other.Method &&
+		c.CaptureFrequencyHz == other.CaptureFrequencyHz &&
+		c.CaptureQueueSize == other.CaptureQueueSize &&
+		c.CaptureBufferSize == other.CaptureBufferSize &&
+		c.Disabled == other.Disabled &&
```

Comment:
[q]: Would we still want a new collector is the capture was enabled, disabled and then re-enabled or would we just want the old one to run?

Reply:
I think we'd want a new collector, since there's that interim period where we don't want it running at all.